### PR TITLE
Simplify Theme

### PR DESCRIPTION
**Pull request by zeitschlag**
We don't support custom themes anymore.

### CODE REVIEW

File: ShareActionExtension/Scene/ShareViewController.swift [hunk 1]
```diff
@@ -69,16 +69,9 @@ extension ShareViewController {
...
-        ThemeService.shared.currentTheme
-            .receive(on: DispatchQueue.main)
-            .sink { [weak self] theme in
-                guard let self = self else { return }
-                self.setupTheme(theme: theme)
-            }
-            .store(in: &disposeBag)
-        
+        setupTheme()
+        ThemeService.shared.apply()
```
Comment (DK1DA) [inline ShareActionExtension/Scene/ShareViewController.swift:73]: Do you know why we have to call `ThemeService.shared.apply()` here? This method is initially invoked in the `AppDelegate`.

**Review by kimar — APPROVED** [review]
LGTM 👍  As said, I'm a fan of removing this.

File: ShareActionExtension/Scene/ShareViewController.swift (hunk 1 above)
Comment (zeitschlag) [reply ShareActionExtension/Scene/ShareViewController.swift:73]: Probably cause it's an extension and I don't know if the AppDelegate gets invoked then?

Comment (DK1DA) [reply ShareActionExtension/Scene/ShareViewController.swift:73]: Ah right, I didn't notice that it's part of the Share Extension. Makes sense then.

**Pull request merged**